### DISCUSSION TITLE
New compatible app found

### DISCUSSION THREAD

**Discussion by lmolnarivan** [darlinghq/darling#1436] [General]
I have found that the macOS version of EdenMath (scientific calculator) works almost completely, the about menu does not work but it is fast, responsive and very usable. Can you include it in the known working software or known partially working software?

**Comment by facekapow** [comment]
Great to hear it's working! I can add it myself, but if you'd like to, you could also add to the docs yourself by adding the info to [this page in the docs](https://github.com/darlinghq/darling-docs/blob/master/src/known-working-software.md) and submitting a pull request. Either way, could you please provide a link to the app's website? If you decide to add the entry to the docs yourself, please include this link in the description.

**Comment by lmolnarivan** [comment]
the link is http://www.edenwaith.com/products/edenmath/
I've tested it on WIP: March 10, 2023 (Update Source 11.5 Public Test) version of darling with the deb package.

**Comment by zeroheure** [comment]
Added, see [#71](/darlinghq/darling-docs/pull/71)
